docs/Flatpak_Build.md

```diff
@@ -0,0 +1,98 @@
+# PyBitmessage Linux flatpak instructions
```

Comment:
Maybe also add this into [index.rst](../../blob/v0.6/docs/index.rst) with mdinclude? Under new section, like
```reST
Build Instructions
------------------
```
Or into [README.md](../../blob/v0.6/README.md)?

Reply:
Okay

Reply:
Quick amendment: include a link to it, not the content, as mdinclude does, I'd mistaken.